crml/cennzx-spot/src/impls.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2019 Centrality Investments Limited
+// This file is part of CENNZnet.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//!
+//! Extra CENNZX-Spot traits + implementations
+//!
+use super::Trait;
+// use cennznet_primitives::FeeExchange;
```

Comment:
Would prefer this commented stuff was removed from this PR (including tests). It can be added back in cleanly in follow ups.